PopPUNK/visualise.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python
+# vim: set fileencoding=<utf-8> :
+# Copyright 2018-2020 John Lees and Nick Croucher
+
+# universal
+import os
+import sys
+# additional
+import numpy as np
+
+# required from v2.1.1 onwards (no mash support)
+import pp_sketchlib
+
+# import poppunk package
+from .__init__ import __version__
+
+from .models import loadClusterFit
+
+from .network import fetchNetwork
+
+from .plot import outputsForMicroreact
+from .plot import outputsForCytoscape
+from .plot import outputsForPhandango
+from .plot import outputsForGrapetree
+from .plot import writeClusterCsv
+
+from .prune_db import prune_distance_matrix
+
+from .sketchlib import readDBParams
+from .sketchlib import getKmersFromReferenceDatabase
+from .sketchlib import addRandom
+
+from .utils import readPickle
+from .utils import setGtThreads
+from .utils import update_distance_matrices
+from .utils import readIsolateTypeFromCsv
+from .utils import joinClusterDicts
+
+#******************************#
+#*                            *#
+#* Command line parsing       *#
+#*                            *#
+#******************************#
+def get_options():
+
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Create visualisations from PopPUNK results',
+                                     prog='poppunk_visualise')
+
+    # input options
+    iGroup = parser.add_argument_group('Input files')
+    iGroup.add_argument('--subset',
+                         help='File with list of sequences to include in visualisation',
```

Comment:
`--subset` may not make sense as a command now this is separate code - maybe just `--input-list`, or something better?